src/index.ts

```diff
@@ -102,3 +102,4 @@ export { default as InvitationResponse } from './businesses/model/InvitationResp
 export { default as SupplierCreateRequest } from './suppliers/model/SupplierCreateRequest';
 export { default as PlaySettings } from './suppliers/model/PlaySettings';
 export { default as Play } from './plays/service/PlaysService';
+export { default as Plays } from './plays/model/Plays';
```

Comment:
There is no need to add `Plays` as a separate model as `Play` have `steps` as an `array` which can be empty as well.
This way you can use play methods as well.